edenia/antelope-tools
# Feat/add links to bp profile page

### Links to BPs Profile Pages

### What does this PR do?

- Resolve #1339
- Resolve #1342
- Resolve #1343
- Resolve #1345
- Remove unnused components

### Steps to test

1. Run the project locally
2. Go to Endpoints page and Geo Location page
3. Check the new links to BPs Profiles
4. Run the project locally with lacchain configuration
5. Check the entities page
6. Check the sitemaps 

### Screenshots

# Entities page

![imagen](https://github.com/edenia/antelope-tools/assets/66583677/b5c4506f-5c35-4eff-80d1-bf783038129b)

# Endpoints page

![imagen](https://github.com/edenia/antelope-tools/assets/66583677/280b7728-c2b5-4029-91b8-374f3c8fd7cb)

# Nodes Distribution

![imagen](https://github.com/edenia/antelope-tools/assets/66583677/79c8fd98-413c-4745-8d13-e83ae09294ae)
![imagen](https://github.com/edenia/antelope-tools/assets/66583677/463151cb-6db0-4c31-8f97-19df7f147fb4)
